### PR TITLE
Add AJAX read/delete for modal and alert page

### LINKED ISSUE #122: Delete Alerts From Modal
At the minute, the modal to show alerts has a link to delete all alerts, but it would be nice to be able to delete single alerts from this same view.

### PR DESCRIPTION
**Pull request by Eldenroot**
- PHP part added
- JS part added
- lang part added
- templates edited

Tested and should be fine, thx Shade and Rocketfood for their ideas :)

Resolves #122

### CODE REVIEW

**Comment by Eldenroot** [comment]
Maybe this one should be removed (not used anymore)
https://github.com/MyBBStuff/MyAlerts/blob/3db3c980c8ab537a194176abcf32e60361abd2c2/inc/languages/english/myalerts.lang.php#L69

File: inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html [hunk 1]
```diff
@@ -1,24 +1,21 @@
-<tr class="alert-row alert--{$alert['alert_code']} {$alert['alert_status']}"
-    id="alert_row_{$alert['id']}">
-    <td class="{$altbg}" width="50" align="center">
+<tr class="alert {$alert['alert_status']} alert--{$alert['alert_code']}"
+    id="alert_row_popup_{$alert['id']}">
```
Comment (euantorano) [inline inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html]: Should this really be `alert_row_popup_`? Isn't this the template for the full page?

File: inc/plugins/myalerts.php [hunk 2]
```diff
@@ -7,7 +7,7 @@
  * @package MyAlerts
  * @author  Euan T. <euan@euantor.com>
  * @license http://opensource.org/licenses/mit-license.php MIT license
- * @version 2.0.3
+ * @version 2.0.4
```
Comment (euantorano) [inline inc/plugins/myalerts.php]: This'll end up getting bumped to 2.0.5, but I might just remove it as it's too say to forget 🤣

Comment (Eldenroot) [reply inc/plugins/myalerts.php]: Removed :)

File: inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html (hunk 1 above)
Comment (Eldenroot) [reply inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html]: You are right, fixed :)

**Comment by Eldenroot** [comment]
@euantorano - please check my reply above (delete lang string)

+ these
**### TO DO/DISCUSS:**
- test "Update alerts" buttons in the alert page (I think there is a small issue on my end)
- maybe replace "text symbols" with "MyBB default icons"

**Comment by Eldenroot** [comment]
Working so far so good _)

**Comment by Eldenroot** [comment]
@euantorano - no problems after 2 months of testing on my live board. Do you plan to merge it and work on 2.0.5 milestone?

**Comment by euantorano** [comment]
Yes, will get round to merging eventually. I've been trying to concentrate on the MyBB core.

**Comment by Eldenroot** [comment]
@euantorano - can you please merge this? :)

**Comment by euantorano** [comment]
Sorry, yes I will do.

**Review by euantorano — APPROVED** [review]

**Pull request merged**